Invalid 'break' statement outside of an enclosing loop
  at: line 5 (<test>)

5 |             (fn() { break })()
6 |                     ^^^^^
